### PR TITLE
Observation/FOUR-13145: Update Login Screen Marketing Text for Winter

### PR DESCRIPTION
**Pull request by agustinbusso**
## Issue & Reproduction Steps
Added changes according to the ticket https://processmaker.atlassian.net/browse/FOUR-13145 and the figma [Platform general Design](https://www.figma.com/proto/sLB2dU5va9iuyKf8Yb8Yjb/Platform-general-Design?page-id=563%3A354&type=design&node-id=563-356&viewport=-8637%2C-3381%2C0.68&t=ot76pTvGabdNWXSt-1&scaling=min-zoom&starting-point-node-id=563%3A356&mode=design)

https://github.com/ProcessMaker/processmaker/assets/90727999/7dacd0c9-0a66-4e64-bfab-9636f6776d70

## Related PRs and Tickets

- https://processmaker.atlassian.net/browse/FOUR-13145
- https://processmaker.atlassian.net/browse/FOUR-13313
- https://processmaker.atlassian.net/browse/FOUR-13322
- [package-auth PR](https://github.com/ProcessMaker/package-auth/pull/95)

ci:next
ci:package-auth:observation/FOUR-13145
ci:deploy

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-7ba73b21bf.engk8s.processmaker.net

File: resources/views/auth/newLogin.blade.php [hunk 1]
```diff
@@ -25,7 +26,14 @@
           @endphp
           @if (!$isMobile)
             <div class="slogan">
-              <img src="/img/slogan.svg" alt="ProcessMaker" />
+              <img src="/img/slogan1.svg" alt="ProcessMaker" />
+              <div class="typewriter-container d-flex align-items-center my-5">
+                <img src="/img/proceC2.svg" class="mr-2 procesC2-icon" alt="ProceC2" />
```
Comment (caleeli) [inline resources/views/auth/newLogin.blade.php:31]: Consider to change the `alt` content to a simple description, could be `ProcessMaker Bot`

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-7ba73b21bf.engk8s.processmaker.net

**Review by caleeli — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-7ba73b21bf.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-7ba73b21bf.engk8s.processmaker.net

**Comment by pmPaulis** [comment]
@agustinbusso 
The font is not the same defined in figma
image
image
Regards,

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-7ba73b21bf.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-7ba73b21bf.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-7ba73b21bf.engk8s.processmaker.net

File: public/images/no-results.svg
Comment (ryancooley) [inline public/images/no-results.svg:1]: Do we know why this and another image are being added in this PR? @agustinbusso

**Review by ryancooley — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-7ba73b21bf.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-7ba73b21bf.engk8s.processmaker.net

**Review by danloa — APPROVED** [review]

**Review by ryancooley — APPROVED** [review]

**Pull request merged**